messages/rule-viewer.md

```diff
@@ -10,3 +10,23 @@
 
 *DRAFT*: %d. %s
 
+# summary.table.id-column
+
+Id
```

Comment:
Violations for a particular run have Ids but I don't think rules should have Ids. Instead can we just have "#" instead of "Id" as the column header when displaying the selected rules in the table.